src/lib/gundb/UserStorageClass.js

```diff
@@ -1342,7 +1366,7 @@ export class UserStorage {
             isPrivate = await this.getFieldPrivacy(field)
           }
 
-          return await this.setProfileField(field, profile[field], isPrivate)
+          return await this.setProfileField(field, profileWithDefaults[field], isPrivate)
```

Comment:
@serdiukov-o-nordwhale i'm merging, but all of this is just another overkill refactoring.
what default values does a profile have? it doesnt. maybe maybe maybe an image placeholder for avatar.
so the whole "defaults" refactoring, might be by the "rules", but in the context of user profile it makes no sense.